packages/e2e/cypress/e2e/app/customDimensions.cy.ts

```diff
@@ -0,0 +1,140 @@
+        body: {
+            name: 'How do payment methods vary across different amount ranges?"',
+            description: 'Payment range by amount',
+            tableName: 'payments',
+            metricQuery: {
+                dimensions: ['payments_payment_method'],
+                metrics: ['orders_total_order_amount'],
+                filters: {},
+                sorts: [
+                    { fieldId: 'orders_total_order_amount', descending: true },
+                ],
+                limit: 500,
+                tableCalculations: [],
+                additionalMetrics: [],
+                customDimensions: [
+                    {
+                        id: 'amount_range',
+                        name: 'amount range',
+                        dimensionId: 'payments_amount',
+                        binType: 'fixed_number',
+                        binNumber: 5,
+                        table: 'payments',
+                    },
+                ],
+            },
+            chartConfig: {
+                type: 'cartesian',
+                config: {
+                    layout: {
+                        flipAxes: false,
+                        xField: 'amount_range',
+                        yField: ['orders_total_order_amount'],
+                    },
+                    eChartsConfig: {
+                        series: [
+                            {
+                                encode: {
+                                    xRef: { field: 'amount_range' },
+                                    yRef: {
+                                        field: 'orders_total_order_amount',
+                                    },
+                                },
+                                type: 'bar',
+                                yAxisIndex: 0,
+                            },
+                        ],
+                    },
+                },
+            },
+            tableConfig: {
+                columnOrder: [
+                    'amount_range',
+                    'orders_total_order_amount',
+                    'payments_payment_method',
+                ],
+            },
+            pivotConfig: {
+                columns: ['payments_payment_method'],
+            },
+        },
+    }).then((r) => {
+        expect(r.status).to.eq(200);
+    });
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export const testCustomDimensions = (projectUuid) => {
+    // Test custom dimension by going into an existing chart with custom dimensions and running the query
+    // This is also used in createProject.cy.ts to test custom dimensions against all warehouses
+    cy.visit(`/projects/${projectUuid}/saved`);
+    cy.contains('How do payment methods vary').click();
+    cy.contains('0-6');
+    cy.contains('6-12');
+};
+describe('Custom dimensions', () => {
+    beforeEach(() => {
+        cy.login();
+    });
+
+    it.only('I can create a custom dimension chart from api', () => {
```

Comment:
forgotten `.only` ?

Reply:
I initially added this to `test` the method locally, without having to createproject, then I forgot to add .skip . I changed this, what do you think ?